www/scripts/codecheckerviewer/ListOfBugs.js

```diff
@@ -209,7 +209,8 @@ function (declare, dom, Deferred, ObjectStore, Store, QueryResults, topic,
         { name : 'Checker name', field : 'checkerId', cellClasses : 'link', width : '50%' },
         { name : 'Severity', field : 'severity' },
         { name : 'Review status', field : 'reviewStatusHtml', cellClasses : 'review-status', width : '25%' },
-        { name : 'Review comment', cellClasses : 'review-comment-message compact', field : 'reviewComment', width : '50%' }
+        { name : 'Review comment', cellClasses : 'review-comment-message compact', field : 'reviewComment', width : '50%' },
+        { name : 'Detection status', field : 'detectionStatus' }
```

Comment:
This makes the detection status appear as `0`, `1` and such, the enum values. There needs to be a method that converts it to some string.